buildroot/package/recovery/init

```diff
@@ -33,6 +33,30 @@ echo 2 >/sys/module/vt/parameters/cur_default
 # Enable syslog
 /etc/init.d/S01logging start > /dev/null
 
+present=`tvservice -l | sed -n -e 's|^Display Number \(.*\),.*$|\1| p'`
+for dev in $present; do
+    fitted=`tvservice -n -v $dev 2>/dev/null`
+    if [[ "$fitted" != "" ]]; then
+        case $dev in
```

Comment:
Given that the values for case 0 and case 2 are the same, perhaps it would make sense to have those set as default outside the loop, and then just switch the case 7 to an `if` instead of a `case` ? :shrug:

Reply:
Maybe that would work too. I wanted to keep it open in case any of the other values were needed too. I did think of reversing the loop nesting to allow different priorities, but I decided the default were sufficient.

Reply:
Fair enough, no harm in keeping it easy-to-extend.